text.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Kindly make sure to rename your file correctly as required you have named it `text.md` instead of `test.md` which is the one to be ignored by git. Kindly fix that.

- [ ] Kindly make sure this `test.md` file which was added in the `gitignore` file is not present in the repository. The reason why we add any file in the `gitgnore` file is that we don't want that file to be pushed on the repository or to not be present in the repo. Kindly make sure to remove it from as it was added in your `gitignore` file.